repo-projects/scenario-tests.proj

```diff
@@ -10,12 +10,12 @@
     <!-- Don't use the updated NuGet.config file that includes the live package feeds when testing source-only as in that configuration
          the Microsoft built packages should be used. -->
     <TestArgs Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' != 'true'">$(TestArgs) /p:TestRestoreConfigFile=$(OriginalNuGetConfigFile)</TestArgs>
-    <UseBootstrapArcade Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' == 'true'">true</UseBootstrapArcade>
+    <UseBootstrapArcade Condition="('$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' == 'true') or '$(SkipScenarioTestsDependencies)' == 'true'">true</UseBootstrapArcade>
   </PropertyGroup>
 
   <!-- In source only test mode, do not depend on other repositories as the scenario tests repo needs to be capable of being built
        by itself in a standalone validation job. -->
-  <ItemGroup Condition="'$(DotNetSourceOnlyTestOnly)' != 'true'">
+  <ItemGroup Condition="'$(DotNetSourceOnlyTestOnly)' != 'true' and '$(SkipScenarioTestsDependencies)' != 'true'">
```

Comment:
I think this can be simplified:

```suggestion
  <ItemGroup Condition="'$(UseBootstrapArcade)' != 'true'">
```

Reply:
> Would `DotNetBuildSourceOnly` always be `true` if `DotNetSourceOnlyTestOnly` is?

Yes, `DotNetSourceOnlyTestOnly` implicitly means a source-only build.

Reply:
Great - will simplify that condition as well.

Reply:
I've simplified the conditions to make them more straightforward, with https://github.com/dotnet/dotnet/pull/327/commits/e1ffe9a44880c9c3ecbc11ecb1e96e7f9ea16a52